.azure-pipelines/azure-pipelines.yml

```diff
@@ -10,15 +10,15 @@ jobs:
         JDK_VERSION: 7
       JDK 8:
         JDK_VERSION: 8
-      JDK 8 with ECJ:
-        JDK_VERSION: 8
-        ECJ: true
       JDK 9:
         JDK_VERSION: 9
       JDK 10:
         JDK_VERSION: 10
       JDK 11:
         JDK_VERSION: 11
+      JDK 11 with ECJ:
```

Comment:
I wonder whether we should use ECJ for more Java versions to run the validation tests for newer language features. What about adding at least JDK 17 to the list (because this is a widely used LTS version)?

Reply:
@marchof I added

```
mvn clean verify -Djdk.version=17 -Dbytecode.version=17 -Decj
mvn clean verify -Djdk.version=19 -Dbytecode.version=19 -Decj
```

LTS and the latest as of now supported by ECJ target version
and updated validation tests accordingly.